test/app-tap/datetime.test.lua

```diff
@@ -2104,4 +2104,26 @@ test:test("Time :set{day = -1} operations", function(test)
     test:is(tostring(ts:set{day = -1}), '1904-03-31T00:00:00Z', '31 March')
 end)
 
+test:test("Parse datetime with specified seconds", function(test)
+    test:plan(11)
+    local dt = date.parse('01', {format = '%S'})
```

Comment:
Could you please check `timestamp` value as well?

Reply:
Sure, added:

```diff
diff --git a/test/app-tap/datetime.test.lua b/test/app-tap/datetime.test.lua
index 657c0dc28a..4cf33d8e06 100755
--- a/test/app-tap/datetime.test.lua
+++ b/test/app-tap/datetime.test.lua
@@ -2105,7 +2105,7 @@ test:test("Time :set{day = -1} operations", function(test)
 end)
 
 test:test("Parse datetime with specified seconds", function(test)
-    test:plan(11)
+    test:plan(12)
     local dt = date.parse('01', {format = '%S'})
     test:is(dt.isdst, false, 'unspecified isdst')
     test:is(dt.nsec, 0, 'unspecified nsec')
@@ -2118,12 +2118,14 @@ test:test("Parse datetime with specified seconds", function(test)
     test:is(dt.month, 1, 'unspecified month')
     test:is(dt.year, 1970, 'unspecified year')
     test:is(dt.hour, 0, 'unspecified hours')
+    test:is(dt.timestamp, 1, 'timestamp')
 end)
 
 test:test("Parse datetime with unspecified seconds", function(test)
-    test:plan(1)
+    test:plan(2)
     local dt = date.parse('01', {format = '%H'})
     test:is(dt.sec, 0, 'uspecified sec')
+    test:is(dt.timestamp, 3600, 'timestamp')
 end)
 
 os.exit(test:check() and 0 or 1)
````